documents/agent-setup.md

```diff
@@ -0,0 +1,112 @@
+| `namespace`           |                     | (optional) Namespace for the current instance.                                                               |
+| `issuerOpts`          |                     | Object containing configuration details and options for credential issuance.                                 |
+|                       | `didOpts`           | Options related to the Decentralized Identifier (DID) of the issuer.                                         |
+|                       | `userPinRequired`   | (optional) Boolean indicating if a user PIN is required for operations.                                      |
+|                       | `cNonceExpiresIn`   | (optional) Number specifying the expiration time for the client nonce in the issuance process.               |
+
+4. **oid4vp_options**: Here, you will store configuration files related to OpenID for Verifiable Presentations (OID4VP).
+
+| Variable           | Sub-Variable          | Description                                                                                             |
+|--------------------|-----------------------|---------------------------------------------------------------------------------------------------------|
+| `definitionId`     |                       | Unique identifier associated with a specific credential definition.                                     |
+| `rpOpts`           |                       | Configuration options for the Relying Party (RP).                                                       |
+|                    | `responseMode`        | (optional) Specifies the method that the RP uses to receive the response from the OpenID Provider.      |
+|                    | `supportedVersions`   | (optional) Lists the versions supported by the RP.                                                      |
+|                    | `sessionManager`      | (optional) An instance of IRPSessionManager from Sphereon's SIOP-OID4VP library managing session state. |
+|                    | `expiresIn`           | (optional) Defines the duration after which the RP's request or session should expire.                  |
+|                    | `eventEmitter`        | (optional) An object used to handle events, enabling asynchronous event-driven programming.             |
+|                    | `didOpts`             | Options related to the Decentralized Identifier (DID) of the verifier.                                  |
+
+5. **presentation_definitions**: The content of this directory is essential for defining how the agent understands and processes presentation requests.
+
+6. **templates**: This directory includes templates for the dynamic generation of content, We're populating the credential using [handlebars](https://handlebarsjs.com/). You can find examples of this usage in this directory. Here is an example of such usage:
```

Comment:
not content, we are talking about how to generate Verifiable Credentials.

Reply:
changed to Verifiable Credentials